src/absil/il.fs

```diff
@@ -4501,7 +4497,7 @@ type ILGlobals with
 
     member this.mkCompilerGeneratedAttribute () = mkILCustomAttribute this (tref_CompilerGeneratedAttribute this, [], [], [])
 
-// Bug 2129. Requests attributes to be added to compiler generated methods 
+// Bug 2129. Requests attributes to be added to compiler generated methods.
```

Comment:
BUG 2129:   refers to an internal Microsoft Bug DB, and given how low the number is it is a really old bug.   We can lose the bug reference, the comment is fine though.